v1/notes/2016-praha-jaro/feedback_treti_lekce.txt

```diff
@@ -0,0 +1,30 @@
+zelené papírky:
++ Vašek mluví dobře a srozumitelně -> neztrácela jsem pozornost!
++ Vysvetlené dostatočne polopatisticky. + za binárnú sústavu a zvieratko (želvičku)
++ Fakt super hodina, moc se mi líbily odbočky, přetažení vůbec nevadilo
++ Domácí úkoly byly výzva, ale zvládla jsem je s nadšením. Velmi jsem se těšila na další lekci. I když lektor odbíhá s vsuvkama, tak mě to baví.
++ Všechno za 1 :)
++ Začátek super, spousta zajímavých inf. navíc. Myslím si, že by se DÚ měly na začátku projít, protože člověk je třeba i zvládne, ale nechápe podstatu (např. já jsem bralajako fakt, že se 'abc' > 'Abc' a jsem ráda, že jsem se dozvěděla proč. Nenapadlo by mě se na to vůbec ptát).
++ Poznámky o příkladech z praxe. Zabíhání do detailů. Vysvětlení některých teoretických základů. Za mě super
++ Skvělý, že Vašek vysvětlil vše v širokém kontextu (teda mat. funkcí bylo možná až moc) a nepřetáhlo se... :)
++ Zajímavé informace. Příjemná rychlost. Srozumitelně podané. Dobře podané extra informace z nesrozumitelných VŠ přednášek.
++ Jsi vtipnej a zajímavě vyprávích, díky, bavilo mě to.
++ Skvělé odbočky, vtipné. Želva je super.
+
+červené papírky
+- Nemám nic, ale musím si na úkoly najít čas, jinak mi ujede vlak. :(
+- Ztratila jsem se ve while a for, musím se na to podívat doma.
+- V domácích úkolech mi debyl jasný 4. úkol. Přesněji: Rozdíly jsou _alespoň_ dva. (nebyla mi jasná volba slov). Pouze u 14. bylo, že se má poslat lektorovi, pokud se má posílat zbytek, tak rozšířit.
+- Bylo to poněkud rychle, ale teda zábavné.
+- Barevné sladění přednášejícího :) (non-native czech speaker)
+- zbytečně dlouho se probíraly úkoly
+- Přetáhl, občas zdlouhavé vysvětlování, spousta informací navíc (ale to není nutně negativní).
+- Plánování času -> nechat víc času na nové věci
+- Timemanagement :)
+
+od kouče
+# úkoly jsem hodně vysvětloval, hlavně špek s 'Aaa' < 'aaa'
+# improvizoval jsem na dané téma, nedržel jsem se striktně podkladů
+# některé zajímavé pasáže z podkladů jsme minuly (python online help, příklady na while), ale jiné přidali (kódování, vnitřní reprezentace proměnných)
+# na while cyklus jsem neudělal žádný příklad, před příští hodinou budu k dispozici dřív a dovysvětlím
```

Comment:
Vypadá to, že se to povedlo;) Uvidíme, jak mi půjde příště Git. Na ten while se taky zeptám na začátku pro ty, které přijdou včas a nebudou se tě moct zeptat. Bylo to zmíněno i ve zpětné vazbě...